src/components/Layout/Sidebar/Link/index.tsx

```diff
@@ -11,11 +11,16 @@ export interface LinkProps {
 }
 
 export const Link: React.FC<LinkProps> = ({ children, onClick, href, isMobile = false }) => {
+  const { pathname } = useLocation();
   if (href[0] === '/') {
     const activeClassName = isMobile ? 'active-mobile-menu-item' : 'active-menu-item';
-
     return (
-      <NavLink to={href} onClick={onClick} activeClassName={activeClassName}>
+      <NavLink
+        to={href}
+        onClick={onClick}
+        activeClassName={activeClassName}
+        isActive={() => href === pathname}
```

Comment:
I think this would prevent sub-pages from marking the parent page as active in the sidebar menu.

You might have missed it, but I already opened a PR to fix this issue: https://github.com/VenusProtocol/venus-protocol-interface/pull/782
I took a different approach and set a conditional `exact` prop on the NavLink so the root route is only marked as active if the current URL is the root and the root only.